modules/push-notifications/src/main/java/com/gluonhq/attach/pushnotifications/impl/AndroidPushNotificationsService.java

```diff
@@ -112,12 +115,21 @@ private GoogleServicesConfiguration readGoogleServicesConfiguration() {
                         if (androidClientInfo != null) {
                             String clientPackageName = androidClientInfo.getString("package_name", "");
                             if (packageName.equals(clientPackageName)) {
+                                clientId = i;
```

Comment:
We should probably read the applicationId in the same way as the apiKey? In other words, move the invocation to `setApplicationId()` to the `if (clientId > -1)` block, like you did for the invocation of `setApiKey()`.

Reply:
Makes sense, fixed